builtin/providers/triton/resource_machine.go

```diff
@@ -108,17 +109,53 @@ func resourceMachine() *schema.Resource {
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
-			"networks": {
```

Comment:
Hi @BrianHicks 

Dropping `networks` will remove backward compatibility. Therefore we need to either:

1. Have a migration in place (e.g. https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/resource_aws_instance_migrate.go)
or 
2. Leave the field in and add the deprecated tag (https://github.com/hashicorp/terraform/blob/master/builtin/providers/cloudstack/resource_cloudstack_nic.go)

If you go for #2 you need to be able to handle the code either way